lerobot/common/datasets/push_dataset_to_hub/_encode_datasets.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+from pathlib import Path
+
```

Comment:
Could you add a docstring at the top of the file to explain what is it useful for and add an example usage (with the command your ran for instance)

Reply:
Done https://github.com/huggingface/lerobot/pull/302/commits/c82c11678b66e11ce763e2474479b324696ca93f